backend/driver.py

```diff
@@ -45,41 +46,37 @@ def root(path):
     else:
         return send_from_directory(app.static_folder, "index.html")
 
+@app.route("/test")
+def verify_backend_alive():
```

Comment:
is this used anywhere?

Reply:
Not really, this is for debugging purposes to ensure the backend URL is reachable. You'd put the URL into your browser and verify the backend is alive and connected

Reply:
I'd say it is needed